apollo-backend/src/main/java/io/logz/apollo/notifications/ApolloNotifications.java

```diff
@@ -71,6 +73,18 @@ public void notify(Deployment.DeploymentStatus status, Deployment deployment) {
                 });
     }
 
+    private boolean filterNotificationByType(Notification notification, Deployment.DeploymentStatus status) {
```

Comment:
I think this method should move to the end of the class after isNotificationInScope method